pom.xml

```diff
@@ -139,7 +139,7 @@
 		<dependency>
 			<groupId>baseCode</groupId>
 			<artifactId>baseCode</artifactId>
-			<version>1.1.6</version>
+			<version>1.1.7</version>
```

Comment:
There's a development version of baseCode that is based on pavlab-starter-parent.